cli/tests/unit/console_test.ts

```diff
@@ -1424,6 +1424,17 @@ unitTest(function consoleTrace(): void {
   });
 });
 
+unitTest(function inspectString(): void {
+  assertEquals(
+    stripColor(Deno.inspect("\0")),
+    `"\\x00"`,
```

Comment:
```suggestion
    String.raw`"\x00"`,
```

Maybe this is slightly less confusing? Just a thought.

Reply:
Hmm, either way is fine; just went with the style of literals and escape tests in the surrounding tests.

Reply:
Leaving as-is since the existing escape asserts are in this form.

Reply:
Makes sense. I think we should either stick to the surrounding style or change them all at once (in a separate PR).